ros2cli/ros2cli/xmlrpc/local_server.py

```diff
@@ -14,7 +14,7 @@
 
 import socket
 import struct
-from xmlrpc.server import SimpleXMLRPCRequestHandler
+from xmlrpc.server import SimpleXMLRPCRequestHandler  # noqa
```

Comment:
do we still need this line here?

Reply:
This one's convenience, so downstream code can get the `SimpleXMLRPCRequestHandler` class from the same module the `LocalXMLRPCServer` is coming from.

Reply:
okay, I would leave a comment saying that we're doing this to re-export SimpleXMLRPCRequestHandler.

Reply:
Comments and documentation added in https://github.com/ros2/ros2cli/pull/652/commits/4ceb6fef7d400f7f3db94dfdd6c9f71f929fcab4.